src/cnv/germline_cnv_calling.md

```diff
@@ -151,7 +151,7 @@ optional arguments:
 ### Example
 ```
 conda activate genomics.py3
-python /VariantCalling/ugvc filter_sample_cnvs \
+python /VariantCalling/ugvc process_cnmops_cnvs \
```

Comment:
can we update the example (and other examples on this readme to ugbio syntax)?

Reply:
I removed this for now, we will add updated documentation prior to releasing all this publicly